main.py

```diff
@@ -180,11 +181,6 @@ def check_configurations(args, pretrain_tasks, target_tasks):
         log.warn("\tMixing training tasks with increasing and decreasing val metrics!")
 
     if args.load_target_train_checkpoint != "none":
-        assert_for_log(
-            os.path.exists(args.load_target_train_checkpoint),
```

Comment:
why are we deleting this?

Reply:
It breaks with wildcards, which we now support. I can fix it, but it'd add a fair bit of code complexity that's probably not worth it.